lib/worker.js

```diff
@@ -85,6 +86,7 @@ Communication.on('JOB', function(job) {
   }
 
   job.Response = new Promise(function(resolve) {
+    const filePath = (eslintignoreDir) ? Path.relative(eslintignoreDir, params.filePath) : params.filePath
```

Comment:
If there is no eslintignore, what would it be relative to? I can't think of any other good way to determine project root. And I don't think absolute paths hurt anything either.